src/main.ts

```diff
@@ -1,16 +1,10 @@
 import * as core from '@actions/core'
-import {wait} from './wait'
+import * as github from '@actions/github'
 
 async function run(): Promise<void> {
   try {
-    const ms: string = core.getInput('milliseconds')
-    core.debug(`Waiting ${ms} milliseconds ...`) // debug is only output if you set the secret `ACTIONS_STEP_DEBUG` to true
-
-    core.debug(new Date().toTimeString())
-    await wait(parseInt(ms, 10))
-    core.debug(new Date().toTimeString())
-
-    core.setOutput('time', new Date().toTimeString())
+    core.info(JSON.stringify(github.context, null, 2))
```

Comment:
`github.context` の中身はこんな感じのオブジェクトだった

<details>

```json
{
  "payload": {
    "action": "created",
    "comment": {
      "author_association": "OWNER",
      "body": "てすと",
      "created_at": "2022-11-23T14:18:46Z",
      "html_url": "https://github.com/mashabow/issue-duplicator-action-test/issues/1#issuecomment-1325142339",
      "id": 1325142339,
      "issue_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1",
      "node_id": "IC_kwDOIfV2Bs5O_BFD",
      "performed_via_github_app": null,
      "reactions": {
        "+1": 0,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 0,
        "url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/comments/1325142339/reactions"
      },
      "updated_at": "2022-11-23T14:18:46Z",
      "url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/comments/1325142339",
      "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/6268183?v=4",
        "events_url": "[https://api.github.com/users/mashabow/events{/privacy}](https://api.github.com/users/mashabow/events%7B/privacy%7D)",
        "followers_url": "https://api.github.com/users/mashabow/followers",
        "following_url": "[https://api.github.com/users/mashabow/following{/other_user}](https://api.github.com/users/mashabow/following%7B/other_user%7D)",
        "gists_url": "[https://api.github.com/users/mashabow/gists{/gist_id}](https://api.github.com/users/mashabow/gists%7B/gist_id%7D)",
        "gravatar_id": "",
        "html_url": "https://github.com/mashabow",
        "id": 6268183,
        "login": "mashabow",
        "node_id": "MDQ6VXNlcjYyNjgxODM=",
        "organizations_url": "https://api.github.com/users/mashabow/orgs",
        "received_events_url": "https://api.github.com/users/mashabow/received_events",
        "repos_url": "https://api.github.com/users/mashabow/repos",
        "site_admin": false,
        "starred_url": "[https://api.github.com/users/mashabow/starred{/owner}{/repo}](https://api.github.com/users/mashabow/starred%7B/owner%7D%7B/repo%7D)",
        "subscriptions_url": "https://api.github.com/users/mashabow/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/mashabow"
      }
    },
    "issue": {
      "active_lock_reason": null,
      "assignee": null,
      "assignees": [],
      "author_association": "OWNER",
      "body": null,
      "closed_at": null,
      "comments": 2,
      "comments_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1/comments",
      "created_at": "2022-11-23T14:02:05Z",
      "events_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1/events",
      "html_url": "https://github.com/mashabow/issue-duplicator-action-test/issues/1",
      "id": 1461818885,
      "labels": [],
      "labels_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1/labels{/name}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1/labels%7B/name%7D)",
      "locked": false,
      "milestone": null,
      "node_id": "I_kwDOIfV2Bs5XIZYF",
      "number": 1,
      "performed_via_github_app": null,
      "reactions": {
        "+1": 0,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 0,
        "url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1/reactions"
      },
      "repository_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test",
      "state": "open",
      "state_reason": null,
      "timeline_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1/timeline",
      "title": "動作確認用",
      "updated_at": "2022-11-23T14:18:46Z",
      "url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/1",
      "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/6268183?v=4",
        "events_url": "[https://api.github.com/users/mashabow/events{/privacy}](https://api.github.com/users/mashabow/events%7B/privacy%7D)",
        "followers_url": "https://api.github.com/users/mashabow/followers",
        "following_url": "[https://api.github.com/users/mashabow/following{/other_user}](https://api.github.com/users/mashabow/following%7B/other_user%7D)",
        "gists_url": "[https://api.github.com/users/mashabow/gists{/gist_id}](https://api.github.com/users/mashabow/gists%7B/gist_id%7D)",
        "gravatar_id": "",
        "html_url": "https://github.com/mashabow",
        "id": 6268183,
        "login": "mashabow",
        "node_id": "MDQ6VXNlcjYyNjgxODM=",
        "organizations_url": "https://api.github.com/users/mashabow/orgs",
        "received_events_url": "https://api.github.com/users/mashabow/received_events",
        "repos_url": "https://api.github.com/users/mashabow/repos",
        "site_admin": false,
        "starred_url": "[https://api.github.com/users/mashabow/starred{/owner}{/repo}](https://api.github.com/users/mashabow/starred%7B/owner%7D%7B/repo%7D)",
        "subscriptions_url": "https://api.github.com/users/mashabow/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/mashabow"
      }
    },
    "repository": {
      "allow_forking": true,
      "archive_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/{archive_format}{/ref}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/%7Barchive_format%7D%7B/ref%7D)",
      "archived": false,
      "assignees_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/assignees{/user}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/assignees%7B/user%7D)",
      "blobs_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/blobs{/sha}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/blobs%7B/sha%7D)",
      "branches_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/branches{/branch}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/branches%7B/branch%7D)",
      "clone_url": "https://github.com/mashabow/issue-duplicator-action-test.git",
      "collaborators_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/collaborators{/collaborator}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/collaborators%7B/collaborator%7D)",
      "comments_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/comments{/number}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/comments%7B/number%7D)",
      "commits_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/commits{/sha}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/commits%7B/sha%7D)",
      "compare_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/compare/{base}...{head}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/compare/%7Bbase%7D...%7Bhead%7D)",
      "contents_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/contents/{+path}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/contents/%7B+path%7D)",
      "contributors_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/contributors",
      "created_at": "2022-11-23T13:51:13Z",
      "default_branch": "main",
      "deployments_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/deployments",
      "description": null,
      "disabled": false,
      "downloads_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/downloads",
      "events_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/events",
      "fork": false,
      "forks": 0,
      "forks_count": 0,
      "forks_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/forks",
      "full_name": "mashabow/issue-duplicator-action-test",
      "git_commits_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/commits{/sha}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/commits%7B/sha%7D)",
      "git_refs_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/refs{/sha}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/refs%7B/sha%7D)",
      "git_tags_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/tags{/sha}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/tags%7B/sha%7D)",
      "git_url": "git://github.com/mashabow/issue-duplicator-action-test.git",
      "has_discussions": false,
      "has_downloads": true,
      "has_issues": true,
      "has_pages": false,
      "has_projects": true,
      "has_wiki": true,
      "homepage": null,
      "hooks_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/hooks",
      "html_url": "https://github.com/mashabow/issue-duplicator-action-test",
      "id": 569734662,
      "is_template": false,
      "issue_comment_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/comments{/number}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/comments%7B/number%7D)",
      "issue_events_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/events{/number}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues/events%7B/number%7D)",
      "issues_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues{/number}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/issues%7B/number%7D)",
      "keys_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/keys{/key_id}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/keys%7B/key_id%7D)",
      "labels_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/labels{/name}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/labels%7B/name%7D)",
      "language": null,
      "languages_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/languages",
      "license": null,
      "merges_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/merges",
      "milestones_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/milestones{/number}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/milestones%7B/number%7D)",
      "mirror_url": null,
      "name": "issue-duplicator-action-test",
      "node_id": "R_kgDOIfV2Bg",
      "notifications_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/notifications{?since](https://api.github.com/repos/mashabow/issue-duplicator-action-test/notifications%7B?since),all,participating}",
      "open_issues": 1,
      "open_issues_count": 1,
      "owner": {
        "avatar_url": "https://avatars.githubusercontent.com/u/6268183?v=4",
        "events_url": "[https://api.github.com/users/mashabow/events{/privacy}](https://api.github.com/users/mashabow/events%7B/privacy%7D)",
        "followers_url": "https://api.github.com/users/mashabow/followers",
        "following_url": "[https://api.github.com/users/mashabow/following{/other_user}](https://api.github.com/users/mashabow/following%7B/other_user%7D)",
        "gists_url": "[https://api.github.com/users/mashabow/gists{/gist_id}](https://api.github.com/users/mashabow/gists%7B/gist_id%7D)",
        "gravatar_id": "",
        "html_url": "https://github.com/mashabow",
        "id": 6268183,
        "login": "mashabow",
        "node_id": "MDQ6VXNlcjYyNjgxODM=",
        "organizations_url": "https://api.github.com/users/mashabow/orgs",
        "received_events_url": "https://api.github.com/users/mashabow/received_events",
        "repos_url": "https://api.github.com/users/mashabow/repos",
        "site_admin": false,
        "starred_url": "[https://api.github.com/users/mashabow/starred{/owner}{/repo}](https://api.github.com/users/mashabow/starred%7B/owner%7D%7B/repo%7D)",
        "subscriptions_url": "https://api.github.com/users/mashabow/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/mashabow"
      },
      "private": true,
      "pulls_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/pulls{/number}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/pulls%7B/number%7D)",
      "pushed_at": "2022-11-23T14:18:30Z",
      "releases_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/releases{/id}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/releases%7B/id%7D)",
      "size": 0,
      "ssh_url": "git@github.com:mashabow/issue-duplicator-action-test.git",
      "stargazers_count": 0,
      "stargazers_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/stargazers",
      "statuses_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/statuses/{sha}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/statuses/%7Bsha%7D)",
      "subscribers_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/subscribers",
      "subscription_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/subscription",
      "svn_url": "https://github.com/mashabow/issue-duplicator-action-test",
      "tags_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/tags",
      "teams_url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test/teams",
      "topics": [],
      "trees_url": "[https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/trees{/sha}](https://api.github.com/repos/mashabow/issue-duplicator-action-test/git/trees%7B/sha%7D)",
      "updated_at": "2022-11-23T13:51:13Z",
      "url": "https://api.github.com/repos/mashabow/issue-duplicator-action-test",
      "visibility": "private",
      "watchers": 0,
      "watchers_count": 0,
      "web_commit_signoff_required": false
    },
    "sender": {
      "avatar_url": "https://avatars.githubusercontent.com/u/6268183?v=4",
      "events_url": "[https://api.github.com/users/mashabow/events{/privacy}](https://api.github.com/users/mashabow/events%7B/privacy%7D)",
      "followers_url": "https://api.github.com/users/mashabow/followers",
      "following_url": "[https://api.github.com/users/mashabow/following{/other_user}](https://api.github.com/users/mashabow/following%7B/other_user%7D)",
      "gists_url": "[https://api.github.com/users/mashabow/gists{/gist_id}](https://api.github.com/users/mashabow/gists%7B/gist_id%7D)",
      "gravatar_id": "",
      "html_url": "https://github.com/mashabow",
      "id": 6268183,
      "login": "mashabow",
      "node_id": "MDQ6VXNlcjYyNjgxODM=",
      "organizations_url": "https://api.github.com/users/mashabow/orgs",
      "received_events_url": "https://api.github.com/users/mashabow/received_events",
      "repos_url": "https://api.github.com/users/mashabow/repos",
      "site_admin": false,
      "starred_url": "[https://api.github.com/users/mashabow/starred{/owner}{/repo}](https://api.github.com/users/mashabow/starred%7B/owner%7D%7B/repo%7D)",
      "subscriptions_url": "https://api.github.com/users/mashabow/subscriptions",
      "type": "User",
      "url": "https://api.github.com/users/mashabow"
    }
  },
  "eventName": "issue_comment",
  "sha": "58d1ac91a5662a833ea6a33163204e521ea08c0e",
  "ref": "refs/heads/main",
  "workflow": "Issue Duplicator",
  "action": "__self",
  "actor": "mashabow",
  "job": "run",
  "runNumber": 2,
  "runId": 3532864997,
  "apiUrl": "https://api.github.com/",
  "serverUrl": "https://github.com/",
  "graphqlUrl": "https://api.github.com/graphql"
}
```

</details>